make/test/JtregNativeHotspot.gmk

```diff
@@ -147,6 +147,12 @@ ifeq ($(call isTargetOs, linux), true)
   NO_FRAMEPOINTER_CFLAGS := -fomit-frame-pointer
 endif
 
+ifeq ($(OPENJDK_TARGET_OS),bsd)
+   PTHREAD=-pthread
```

Comment:
This should really be a LIB_PTHREAD in spec.gmk, which is set by configure. And we should do a search&replace on all literal instances of `-lpthread` in the code base. I recommend doing that as a separate PR, prior to trying to integrate this. (In fact, that might even be an excellent starter PR for you to get your feet wet on the JDK process.)

Reply:
I'll bite on that one :)